md/workspaces-and-repositories.md

```diff
@@ -102,12 +99,17 @@ Bonita studio Git integration is based on th EGit eclipse plugin.
 
 Git command available in Studio interface:
 
-* **Share with Git**  
-This action connect the current repository to Git and share it on a remote.
-To configure the remote see the following [Egit userguide](http://wiki.eclipse.org/EGit/User_Guide#Working_with_remote_Repositories) or the [Share on GitHub howto](share-a-repository-on-github.md).
+ * **Share with Git**  
+This action connects the current repository to Git and shares it on a remote.
+To configure the remote, see the following [Egit userguide](http://wiki.eclipse.org/EGit/User_Guide#Working_with_remote_Repositories) or the [Share on GitHub howto](share-a-repository-on-github.md).
 
-* **Clone**  
-Create an new Studio repository from an existing Git repository (must contains a proper Bonita project). If the remote repository version is lower than the Studio, a migration will be applied on the cloned repository. Be carefull before pushing a migration on the remote. All contributors will have to use the proper Studio version.
+ * **Clone**  
+Create a new Studio repository from an existing Git repository (that must contain a proper Bonita project). If the remote repository version is lower than the Studio, a migration will be applied on the cloned repository. Be careful before pushing a migrated repository back to the remote: all contributors will have to use the proper Studio version.
+If you used Bonita on Git before Bonita 7.7.0, you might want to clone it directly from the Studio. For that, your project on github must be Bonita compliant:
```

Comment:
Do we support that ?
I'm wondering if we shouldn't say that there is no guaranty (Or is there ?) and to clone manually if it doesn't work

Reply:
Agree, depending on what has been comitted it could work or not, even if the .project is present. 
It has been requested by pablo and Enrico apparently.

Reply:
OK. Thanks guys. I will add this warning line.